CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ### citus v11.1.2 (September 30, 2022) ###
 
+* Adds support for PostgreSQL 15 RC1
```

Comment:
For the sake of consistency,

```suggestion
* Adds support for PostgreSQL 15rc1
```

We had an item like the following for `v11.1.0`
```md
* Adds support for PostgreSQL 15beta4
```